grype/db/v6/distribution/status.go

```diff
@@ -0,0 +1,16 @@
+package distribution
+
+type Status string
+
+const LifecycleStatus = StatusActive
+
+const (
+	// StatusActive indicates the database is actively being maintained and distributed
+	StatusActive Status = "active"
+
+	// StatusDeprecated indicates the database is still being distributed but is approaching end of life. Upgrade grype to avoid future disruptions.
+	StatusDeprecated Status = "deprecated"
```

Comment:
This isn't used anywhere, but when downloading updates if the `latest.json` says we're updating a deprecated or eol schema version, we should `log.Warn` and partybus.Notify people that they need to upgrade.